python/cuml/svm/svc.pyx

```diff
@@ -244,6 +278,18 @@ class SVC(SVMBase, ClassifierMixin):
         self._set_output_type(X)
         self._set_target_dtype(y)
 
+        if self.probability:
+            params = self.get_params()
+            params["probability"] = False
+            X, _, _, _, _ = input_to_host_array(X)
+            y, _, _, _, _ = input_to_host_array(y)
+            with using_output_type('numpy'):
+                self.prob_svc = CalibratedClassifierCV(SVC(**params), cv=5,
```

Comment:
I think linking the issue you created for a gpu based CalClass would be good to have here (if its somewhere else and I missed it, feel free to ignore my comment) https://github.com/rapidsai/cuml/issues/2608

Reply:
Good point, I have added a comment with a link to the issue